### ISSUE TITLE
How to close context?

### ISSUE THREAD

**Issue by kpeu3i** [hajimehoshi/oto#149]
I'm trying to switch to a new version (v2) of the library but seems there is no way to recreate context now. But I see this option documented here https://github.com/hajimehoshi/oto/blob/main/driver.go#L26? What I missed?  Context has Suspend and Resume methods, are they do the job?

**Comment by hajimehoshi** [comment]
I have removed `Close` from `Context` as of v2. Then the context will live until the process dies.

Suspend and Resume works if you want to pause audio temporarily.

**Comment by kpeu3i** [comment]
hmm, but what if having different contexts created with the different args (sampleRat, channelNum, bitDepthInBytes) might be necessary during the app lifecycle?

**Comment by hajimehoshi** [comment]
Yeah, or creating a process, using oto there, and closing it would be necessary.

I welcome contributions to add `(*Context).Close` though I think it is not easy to implement this correctly in all the platforms...

**Comment by kpeu3i** [comment]
Got it, thank you for the quick response! Seems like I can reuse the same context in my app...

**Comment by hajimehoshi** [comment]
Another suggestion is to resample your audio source, which can be a quite heavy task.

**Comment by d3xMachina** [comment]
This prevent my computer going to standby after using speaker.Init().
You can check that with the command "powercfg /requests" in a command line launched with admin rights : there will be a driver entry under SYSTEM that will only go away when the application using this package is closed. Calling speaker.Close() or speaker.Clear() doesn't help either.
